mail-client/alot/alot-0.8.1.ebuild

```diff
@@ -18,17 +18,16 @@ IUSE="doc test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
-	>=app-crypt/gpgme-1.9.0[python,${PYTHON_USEDEP}]
-	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	app-crypt/gpgme[python,${PYTHON_USEDEP}]
```

Comment:
There is no reason to touch this ebuild at all. All these changes require a revbump. Just leave it like it is.

Reply:
Ok I keep that change for an other PR.